frontend/src/utils/help.ts

```diff
@@ -0,0 +1,31 @@
+import { RouteLocationNormalized } from "vue-router";
+import { useUIStateStore, useHelpStore } from "@/store";
+
+export const helpOnRouteChange = async (to: RouteLocationNormalized) => {
+  const { name } = to;
+  const uiStateStore = useUIStateStore();
+  const helpStore = useHelpStore();
+  const res = await fetch("/help/routeMap.json");
+  const routeMap = await res.json();
+
+  if (routeMap[name as keyof typeof routeMap]) {
+    if (
+      !uiStateStore.getIntroStateByKey(
+        `guide.${routeMap[name as keyof typeof routeMap]}`
+      )
+    ) {
+      if (helpStore.timer) {
```

Comment:
`helpStore.timer` appears to be used only in this function, so we can define it as a variable in this file.